### PR TITLE
Add link to trace compass to readme

### PR DESCRIPTION
**Pull request by planger**

### CODE REVIEW

**Review by MatthewKhouzam — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For more information, follow the links to the components below or visit the [pro
   * [CDT GDB DAP Adapter](https://github.com/eclipse-cdt-cloud/cdt-gdb-adapter) and its [VS Code extension](https://github.com/eclipse-cdt-cloud/cdt-gdb-vscode): Debug adapter for GDB
   * [CDT Amalgamator](https://github.com/eclipse-cdt-cloud/cdt-amalgamator): Debug adapter allowing common control over multiple debug adapters in VS Code
   * [Clangd Contexts](https://github.com/eclipse-cdt-cloud/clangd-contexts): API for management of clangd configuration files in C/C++ projects using contexts.
-  * Eclipse Trace Compass Cloud (🔧 in progress)
+  * [Eclipse Trace Compass Cloud](https://github.com/eclipse-cdt-cloud/theia-trace-extension): Trace viewer for Eclipse Theia
```
Comment (MatthewKhouzam) [inline README.md:22]: Should it be Eclipse Trace Compass.cloud?

Comment (planger) [reply README.md:22]: Yes :+1: I think that'd be better

**Comment by bhufmann** [comment]
Looks good to me too. Thanks!

**Pull request merged**